components/home/Challenge.tsx

```diff
@@ -69,17 +69,23 @@ export function Challenge({
         style={{ backgroundColor: "transparent" }}
       >
         <TouchableOpacity
+          testID="challenge-touchable"
           onPress={() => setIsOpen(!isOpen)}
           activeOpacity={0.8}
         >
           <ThemedView style={[styles.challenge]}>
             <Image
-              source={require("@/assets/images/challenge2.png")}
+              testID="challenge-image"
+              source={
+                challengeDB.image_id
+                  ? { uri: challengeDB.image_id }
+                  : require("@/assets/images/challenge1.png")
+              }
```

Comment:
The displayed image in the case the post doesn't have one should be more specific (ex : a black image, an image with error message, etc)